### DISCUSSION TITLE
My locally hosted server shows LGSM Valheim server as ONLINE but I can't connect to its public IP

### DISCUSSION THREAD

**Discussion by Sidekicksfordinner** [GameServerManagers/LinuxGSM#3345] [Q&A] [answered]
I've been trying to set this server up all day and I'm just about at my wits end. I hope I'm doing something noobish and someone can point it out to me...

https://termbin.com/3571

I've installed Ubuntu 20.10 server on a baremetal machine in my house, and on that I've installed LinuxGSM for Valheim. The Valheim server shows status: ONLINE but I can't connect to it from the public IP address. I've added the public IP to my Steam favorite servers but I can't connect, and the page shows it as "not responding".

I've added port forwarding rules to my EdgeRouter along the lines of:
Original port: 2456-2458
forward to address: [LAN IP]:[2456-2458]

Any help is mucho appreciated!

**Comment by wldwalker** [comment, marked as answer]
That's normal. You need to enable NAT loopback in your router, if it even supports it. So, for you, just use your internal Ip to connect, your friends however will use the external IP.

**Reply by dgibbs64** [reply]
Side note I have changed the word online to started. As there is often confusion by what this means 👍

**Reply by Sidekicksfordinner** [reply]
Wow it's actually working at the local IP LOL thank you!
